src/api/search.js

```diff
@@ -30,6 +33,21 @@ export default class extends Base {
         }
 
         Hooks.emit('search-quick-remote', query);
-        this.makeRequest(url, 'GET', paramsArg, false, callbackArg);
+
+        if (activeRequest === true) {
```

Comment:
don't you want abort to happen as early as possible?

Reply:
Do we still want to emit search-quick-remote if the request will be cancelled?

Reply:
I think so. The request won't always be cancelled. It will only happen when a new request is triggered before a response is received. In testing it was difficult to do but maybe it gets easier with a large catalog.

Reply:
Tbf though, i'm not sure how people use the `Hooks.emit()` functionality so how big of a problem it would be to receive event that could end up getting cancelled.